ci/install_google_cloud_sdk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-# Note: We now install and update the Google Cloud SDK in .travis.yml.
+# Note: We now install and update the Google Cloud SDK in ci.yml.
```

Comment:
Do we need this file or this whole directory now?  I don't see how it is used.  I think it might not have even been used under travis.

Reply:
This file was used for automatic staging deployment for Travis. I thought we would keep it just in case we want to deploy staging instances again. wdyt

Reply:
OK, it would be good to eventually have automatic deployment to staging, especially if we had automated system tests.  I assume that we could limit such deployments to only PRs that are sent by users on a short allow-list.   I am not sure that we would need this particular file, but we can certainly keep it around.  Or, if we delete it, it would not be hard to undelete.

Reply:
Yea I will delete this for now